### DISCUSSION TITLE
Next Quickstarts & SDKs?

### DISCUSSION THREAD

**Discussion by dickhardt** [hellocoop/hello.dev#53] [Ideas]
We are looking for feedback on which Quickstarts and SDKs to create next. 

Server side `express`, `fastify` are pretty straightforward as they are similar to the Next.js [SDK](https://www.npmjs.com/package/@hellocoop/nextjs) and [starter](https://github.com/hellocoop/hello-nextjs-starter) and will build upon the [`@hellocoop/core`](https://www.npmjs.com/package/@hellocoop/core) package

What other platforms would you like to see?

**Comment by jonsage** [comment]
Astro please

**Comment by dickhardt** [comment]
@jonsage -- happy to do that if we have someone that knows Astro to guide and review. Would you be up for that?

I took a look at https://www.npmjs.com/package/@astro-auth/providers -- the docs link redirects to a phishing site -- not a great start to checking out the Astro code base

**Reply by jonsage** [reply]
Yeah, that is known to the Astro team but not sure anything can be done about it - I'm not particular good with Astro or how it works, I just want more options for auth.

Last I checked the closest to an official (although not official) auth solution was https://www.npmjs.com/package/auth-astro which is basically a provider for `next-auth.js` (now just `auth.js` - https://authjs.dev/)

**Comment by carrete** [comment]
> What other platforms would you like to see?

https://github.com/ring-clojure/ring

**Reply by dickhardt** [reply]
thanks -- @carrete  are you interested in helping?

**Reply by carrete** [reply]
Sure. I'd be happy to help, @dickhardt

**Reply by mscurtescu** [reply]
@carrete I reached out on Mastodon, let's connect and figure what we can do for Clojure / Ring.

**Comment by anishanilkumar** [comment]
Drupal

**Comment by romw314** [comment]
Auth.js please. It can be used with other framework and already has Google, Apple, GitHub, etc. login support, Hellō would be very good.
